src/sage/rings/fraction_field.py

```diff
@@ -557,6 +557,30 @@ def is_exact(self):
         """
         return self.ring().is_exact()
 
+    def _convert_from_finite_precision_laurent_series(self, x):
+        """
+        Construct an element of this fraction field approximating a Laurent series.
+
+        INPUT:
+
+        - ``x`` -- a Laurent series, must have finite precision
+
+        OUTPUT: Element of ``self``
+
+        This internal method should not be used directly, use :meth:`__call__` instead,
+        which will delegates to :meth:`_element_constructor_`. There are some tests there.
+
+        .. NOTE::
+
+            Uses the algorithm described in `<https://mathoverflow.net/a/14874>`_.
+            This may be changed to use Berlekamp--Massey algorithm or something else
+            to compute Padé approximant in the future.
+        """
+        integral_part, fractional_part = self(x.truncate(1)), x.truncate_neg(1)
```

Comment:
```suggestion
        integral_part = self(x.truncate(1)
        fractional_part = x.truncate_neg(1)
```
I find this easier to understand (and perhaps even PEP8 requires it?).